src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -40,6 +41,26 @@ public function __construct($statusMetadataExtractor = null)
         $this->statusMetadataExtractor = $statusMetadataExtractor ?: new StatusMetadataExtractor();
     }
 
+    /**
+     * Returns the customer ID information of the provided request.
+     *
+     * @param Message $request the request to get its customer ID
+     * @return string the customer ID information
```

Comment:
You may want to return an int instead of string (using intval)

Reply:
We can't. We returns a string when no customer ID can be found.